backend/integrationtest/HyvaksymisEsitys/listaaTiedostot.test.ts

```diff
@@ -180,6 +181,7 @@ describe("Hyväksymisesityksen tiedostojen listaaminen (aineistolinkin katselu)"
         hyvaksyja: "theadminuid",
         hyvaksymisPaiva: "2022-01-01",
       },
+      aineistoHandledAt: "2022-01-01",
     };
     await insertProjektiToDB(projektiInDB);
     // Testataan, saadaanko uudet tiedostot
```

Comment:
Koska aineistoHandledAt puuttui näistä, BE:n testit ei saanut kiinni yhtä bugia, minkä korjasin tuossa alla.